app/api/v1/endpoints/seasonal_forecast_pdf/router.py

```diff
@@ -0,0 +1,25 @@
+from typing import OrderedDict
+from fastapi import APIRouter, HTTPException
+from app.core.webScrape.webscraper import scrape_webpage, mw_seasonal_forecast_list_url
+
+from .schema import (
+    SeasonalForecastPDFParameters,
+)
+
+router = APIRouter()
+
+
+@router.post("/")
+async def get_seasonal_forecast_list(
+    params: SeasonalForecastPDFParameters,
+)-> OrderedDict:
+    if params.country == "mw":        
+        district_links = scrape_webpage(mw_seasonal_forecast_list_url)
+        # Check if link_text exists in the dictionary
+        if params.district in district_links:
+            return {"pdf_url": district_links[params.district ]}
```

Comment:
nit(blocking) - I feel like the endpoint should probably be returning the actual pdf file instead of a link, particularly as we may end up serving these directly from the storage bucket (instead of public websites) which we may not want/be able to provide access links to.

I haven't tested myself, but it might be as simple as wrapping the pdf url in a fastapi [File Response](https://fastapi.tiangolo.com/advanced/custom-response/#fileresponse)